### ISSUE TITLE
can't search XboxSeriesS Controller buletooth device

### ISSUE THREAD

**Issue by Ekkone** [hugbug/ev3#2]
XboxSeriesS Controller can't use this way, can you test that?

**Comment by hugbug** [comment]
I can confirm the third generation Xbox wireless controller (Series S and X) isn't working. I've tested two of them and also an elite 2 controller. None of the three worked. Retested with the Xbox One controller and it appeared immediately in the device list on ev3 and connected successfully.

I've also noticed these newer controllers cannot connect to my old MacBook (from year 2010) whereas Xbox One connects successfully. In fact the newer controllers do not appear in the list of Bluetooth devices on Mac at all. All controllers connect successfully to a much more recent iPad.

My guess is all this is related to the Bluetooth version. According to Wikipedia page on Xbox wireless controller, the Xbox one is the second generation with Bluetooth 4.0 and Series S/X is the third generation with Bluetooth LE.
